Voycar.Api.Web/Features/Reservations/Endpoints/Delete/Personal/Endpoint.cs

```diff
@@ -0,0 +1,68 @@
+namespace Voycar.Api.Web.Features.Reservations.Endpoints.Delete.Personal;
+
+using Entities;
+using Repository;
+using Users.Repository;
+
+
+public class Endpoint : Endpoint<Request, Results<Ok, BadRequest<string>>>
```

Comment:
```suggestion
public class Endpoint : Endpoint<Request>
```
`BadRequest` haben wir glaube ich bisher generell nicht als Response angegeben, muss auch nicht, kann Fast Endpoints so. Und spezifisch `BadRequest<string>` stimmt auch nicht mehr.